src/fides/core/config/security_settings.py

```diff
@@ -16,42 +15,93 @@
+    )
+    identity_verification_attempt_limit: int = Field(default=3, description="")
+    oauth_root_client_id: str = Field(
+        default="",
+        description="The value used to identify the Fides application root API client.",
+    )
+    oauth_root_client_secret: str = Field(
+        default="",
+        description="The secret value used to authenticate the Fides application root API client.",
+    )
+    oauth_root_client_secret_hash: Optional[Tuple] = Field(
+        default=None,
+        description="Automatically generated by Fides, and represents a hashed value of the oauth_root_client_secret.",
+    )
+    oauth_access_token_expire_minutes: int = Field(
+        default=11520,
+        description="The time in minutes for which Fides API tokens will be valid. Default value is equal to 8 days.",
+    )
+    oauth_client_id_length_bytes: int = Field(
+        default=16,
+        description="Sets desired length in bytes of generated client id used for oauth.",
+    )
+    oauth_client_secret_length_bytes: int = Field(
+        default=16,
+        description="Sets desired length in bytes of generated client secret used for oauth.",
+    )
+    parent_server_password: Optional[str] = Field(
+        default=None,
+        description="When using a parent/child Fides deployment, this password will be used by the child server to access the parent server.",
+    )
+    parent_server_username: Optional[str] = Field(
+        default=None,
+        description="When using a parent/child Fides deployment, this username will be used by the child server to access the parent server.",
+    )
+    rate_limit_prefix: str = Field(
+        default="fides-",
+        description="The prefix given to keys in the Redis cache used by the rate limiter.",
+    )
+    request_rate_limit: str = Field(
+        default="1000/minute",
+        description="The number of requests from a single IP address allowed to hit an endpoint within a rolling 60 second period.",
+    )
+    root_user_scopes: List[str] = Field(
+        default=SCOPE_REGISTRY,
+        description="The list of scopes that are given to the root user.",
+    )
+    root_password: Optional[str] = Field(
+        default=None,
+        description="If set, this can be used in conjunction with root_username to log in without first creating a user in the database.",
+    )
+
+    root_username: Optional[str] = Field(
+        default=None,
+        description="If set, this can be used in conjunction with root_password to log in without first creating a user in the database.",
+    )
+    subject_request_download_link_ttl_seconds: int = Field(
+        default=432000,
+        description="The number of seconds that a pre-signed download URL when using S3 storage will be valid. The default is equal to 5 days.",
```

Comment:
Nit and not likely a change in this PR but it feels inconsistent to use minutes above (i.e. `oauth_access_token_expire_minutes`) and seconds here to describe a relatively similar timeframe

Reply:
I hugely agree here...every field should include the unit of measurement and also use them consistently, however making this change now is difficult because it would be a breaking change